types/mempool/sender_nonce.go

```diff
@@ -183,7 +180,6 @@ func (snm *senderNonceMempool) Remove(tx sdk.Tx) error {
 	_, found = snm.existingTx[key]
 	if found {
 		delete(snm.existingTx, key)
```

Comment:
This is a nit but now that we are just using the map entirely, the cooler thing is that we can now just invoke `delete(snm.existingTx, key)` whether the key exists or not.

Reply:
cool will change soon